### DISCUSSION TITLE
Dir Tree

### DISCUSSION THREAD

**Discussion by Divelix** [squidfunk/mkdocs-material#6902] [Ideas]
Does MkDocs have dir tree display feature like [this](https://github.com/k4a-l/obsidian-dirtreeist) Obsidian plugin? I couldn't find that after 10 mins of googling. If not, it would be a cool feature to have.

**Comment by kamilkrzyskow** [comment]
Hello @Divelix,
I don't think there is anything like this 🤔 Also this would be a markdown extension that processes code blocks in a specific way to display the file structure based on the contents. I didn't find anything of that sort outside the MkDocs bubble either.

You could raise a feature request to https://github.com/facelessuser/pymdown-extensions or to https://github.com/Python-Markdown/markdown, as this is more aligned with their scope than the theme's, though recently I've noticed that squidfunk started implementing them (for example https://github.com/squidfunk/mkdocs-material/issues/6704)
